OverflowError: Python int too large to convert to C unsigned long

On Windows 64 bit, i'm getting an `OverFlowError` when creating a language using a custom binding:

```python
from tree_sitter import Language

from . import _binding

LANGUAGE = Language(_binding.language())
```

```
    LANGUAGE = Language(_binding.language())
E   OverflowError: Python int too large to convert to C unsigned long
```

This is caused by commit `14e7893` (refactor: prefer unsigned long over size_t)

You cannot use an unsigned long for a pointer, on Win64 an unsigned long is 32-bit! Use `uintptr_t` instead!

--- a/tree_sitter/binding/language.c
+++ b/tree_sitter/binding/language.c
@@ -9,7 +9,7 @@ int language_init(Language *self, PyObject *args, PyObject *Py_UNUSED(kwargs)) {
     if (PyCapsule_CheckExact(language)) {
         self->language = PyCapsule_GetPointer(language, "tree_sitter.Language");
     } else {
-        Py_uintptr_t language_id = PyLong_AsUnsignedLong(language);
+        Py_uintptr_t language_id = PyLong_AsSize_t(language);
         if (language_id == 0 || (language_id % sizeof(TSLanguage *)) != 0) {
             if (!PyErr_Occurred()) {
                 PyErr_SetString(PyExc_ValueError, "invalid language ID");

